appveyor.yml

```diff
@@ -41,6 +43,12 @@ environment:
         NUMPY_VERSION: "1.10"
         ASTROPY_VERSION: "lts"
 
+      - PYTHON_VERSION: "3.5"
+        NUMPY_VERSION: "stable"
```

Comment:
I think the appveyor builds are getting rather long, and since there is only one process for the whole astropy family, it would be nice to cut back a bit. This build basically tests among others that numpy and astropy understands and works with "stable", so maybe the builds in L19 and L38 can be removed?